engine/render/src/render/render.cpp

```diff
@@ -403,8 +403,16 @@ namespace dmRender
         for( uint32_t i = 0; i < num_ranges; ++i)
         {
             RenderListRange& range = ranges[i];
-            if ( (range.m_TagMask & tag_mask) != tag_mask )
+
+            MaterialTagList taglist;
+            dmRender::GetMaterialTagList(context, range.m_TagListKey, &taglist);
+
+            range.m_Skip = 0;
```

Comment:
The same list of ranges is done throughout the frame, so we need to reset the skip flag for each draw call.